e2e/test/MethodE2ETests.cs

```diff
@@ -102,6 +105,28 @@ public async Task Method_DeviceReceivesMethodAndResponseWithDefaultMethodHandler
             await SendMethodAndRespond(Client.TransportType.Amqp_WebSocket_Only, SetDeviceReceiveMethodDefaultHandler).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        public async Task Method_ServiceSendsMethodThroughProxyWithDefaultTimeout()
+        {
+            ServiceClientTransportSettings serviceClientTransportSettings = new ServiceClientTransportSettings()
+            {
+                HttpProxy = new WebProxy(Configuration.IoTHub.ProxyServerAddress)
+            };
+
+            await SendMethodAndRespond(Client.TransportType.Mqtt_Tcp_Only, SetDeviceReceiveMethod, serviceClientTransportSettings).ConfigureAwait(false);
+        }
+
+        [TestMethod]
+        public async Task Method_ServiceSendsMethodThroughProxyWithCustomTimeout()
+        {
+            ServiceClientTransportSettings serviceClientTransportSettings = new ServiceClientTransportSettings()
```

Comment:
When you use an initializer list and don't send ctor params, you may leave out the parenthesis.
Also, when the type is obvious, we prefer `var`.

```
var serviceClientTransportSettings = new ServiceClientTransportSettings
{
    HttpProxy = ...,
};
```

Reply:
Sure thing